stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -668,6 +686,13 @@ spec:
         emptyDir:
           sizeLimit: {{ .Values.artifactory.persistence.size }}
   {{- end }}
+      {{- if .Values.metrics.enabled }}
+      - name: artifactory-logs
```

Comment:
What happens if you start off without `metrics.enabled` and decide you want to enable it later on? From when I understand, due to the conditional volume, all the logs will go away. I think this volume should not be conditioned and should always be created

Reply:
I'm not actually sure, I'll test it out now and confirm but I suspect you're correct.

Reply:
I don't think this is a problem because when you apply the Helm update to set `metrics.enabled` to true, the pods will get destroyed and recreated.  This is going to cause the volume to be destroyed as well since it is of type `emptyDir` which has the same lifetime of the pod.  I think a second confirmation would be good though.

Reply:
Well, I think `emptyDir` is problematic. You want logs to last over time, and the lifespan of an `emptyDir` volume is the same as the pod's. For my other point, Consider a situation where you already have an Artifactory installation running, it already has some logs on the PVC, and now when you configure `metrics.enabled=true` the previously created logs will be overwritten by a new (empty) volume. That will be a problem. IMO you shouldn't use a volume specifically for logs and just use the main volume we already have. You can simply mount It in the fluentd container

Reply:
Pushed up a fixed version where I'm mounting the main volume instead.  Thanks for the comment.